desktop/dashboard.md

```diff
@@ -220,3 +224,21 @@ To interact with remote repositories:
     The **View in Hub** option opens the Docker Hub page and displays detailed information about the image, such as the OS architecture, size of the image, the date when the image was pushed, and a list of the image layers.
 
     If you have subscribed to a Pro or a Team plan and have enabled [Vulnerability Scanning](../docker-hub/vulnerability-scanning.md) the Docker Hub page also displays a summary of the vulnerability scan report and provides detailed information about the vulnerabilities identified.
+
+## Manage volumes
+
+The **Volumes** view in Docker Dashboard enables you to easily create and delete volumes and see which ones are being used. If you are a developer with a Pro or a Team subscription, you can also see which container is using a specific volume and explore the files and folders in your volumes.
```

Comment:
nit: should we link «a Pro or a Team subscription» to the pricing matrix or somewhere similar? I’m not sure if/how it’s done elsewhere

Reply:
Good point. Let me add a note

Reply:
Updated